.github/workflows/lint.yml

```diff
@@ -13,4 +13,4 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Run golangci-lint
-      uses: actions-contrib/golangci-lint@v1
+      uses: matoous/golangci-lint-action@v1.23.3
```

Comment:
It seems that the official action is to be found here https://github.com/golangci/golangci-lint-action

matoous/golangci-lint-action appears to be deprecated

Reply:
Sure.  I have updated this in the latest push.  Thanks!